pennylane/decomposition/decomposition_graph.py

```diff
@@ -491,8 +491,7 @@ def solve(self, num_work_wires: int | None = 0, lazy=True) -> DecompGraphSolutio
             op_names = {op_node.op.name for op_node in unsolved_ops}
             warnings.warn(
                 f"The graph-based decomposition system is unable to find a decomposition for "
-                f"{op_names} to the target gate set {set(self._gate_set_weights)}. The default "
-                "decomposition (op.decomposition()) for these operators will be used instead.",
+                f"{op_names} to the target gate set {set(self._gate_set_weights)}.",
```

Comment:
Silly question: why are tests passing if the error message for this changed?

Reply:
Because we only test with matching [the first part of the error message](https://github.com/PennyLaneAI/pennylane/blob/4a78cc7bc6cd7364693696f66eb2e2ad8319ab9b/tests/transforms/test_decompose_transform_graph.py#L307).

Reply:
because I was lazy and I didn't want to type out the full message lol